src/connection.js

```diff
@@ -1245,6 +1286,7 @@ Connection.prototype.STATE = {
     name: 'SentClientRequest',
     exit: function() {
       this.clearRequestTimer();
+      this.tokenStreamParser.resume();
```

Comment:
It's more complicated than that.

1. As long as Connection.execSql() has not yet been called, the request does not know the connection.
2. When the request is cancelled or time out, we have to resume the token parser transform, but the request can still remain in the paused state.

So the paused state of the Request objects are independent of the paused state of the token parser transform.